apps/sharemyhealth/resources.py

```diff
@@ -0,0 +1,9 @@
+from social_django.models import UserSocialAuth
+
+
+class Resource(object):
+    """A python wrapper around the social_django.models.UserSocialAuth class."""
+    model_class = UserSocialAuth
+
+    def filter_by_user_and_provider(self, user, provider):
```

Comment:
`provider` can be removed, should be specific for a class. A future extension could allow a more generic implementation class with a provider name specified by the string stored in the DB.